SpriterDotNet/Parser/XmlSpriterParser.cs

```diff
@@ -24,7 +26,14 @@ public Spriter Parse(string data)
 
         public bool CanParse(string data)
         {
-            return data.StartsWith(XmlStart);
+            return data != null && data.StartsWith(XmlStart);
+        }
+
+        private static string FixBadNanValue(string data)
+        {
+            var nanRegex = new Regex(@"(a)=""nan""");
+            data = nanRegex.Replace(data, @"$1=""NaN""");
```

Comment:
Ah k, makes sense. Wasn't completely aware of the details of the bug, just knew that "nan" didn't work right. But it will be documented in the test when I fix it.

As far as removing it later, wouldn't that just be opening up a chance for the bug to reappear later, as well as cause issues for people who try to open files that were saved when the bug was present?